### ISSUE TITLE
pigeon_build_runner doesn't work with latest version of pigeon

### ISSUE THREAD

**Issue by JaseElder** [rotorgames/pigeon_build_runner#2]
I'm getting this error on `flutter pub add`:

```
flutter pub add dev:pigeon_build_runner
Because every version of pigeon_build_runner depends on pigeon ^10.0.0 and tomoto depends on pigeon ^11.0.1, pigeon_build_runner is forbidden.
So, because tomoto depends on pigeon_build_runner any, version solving failed.
```

**Comment by rotorgames** [comment]
Fixed in https://github.com/rotorgames/pigeon_build_runner/commit/8ff4ee1d11f122a0e917faaebab95e91e113e741
Available from [v1.0.8](https://pub.dev/packages/pigeon_build_runner/versions/1.0.8)

**Comment by JaseElder** [comment]
@rotorgames Thank you!
